e2e/test/prerequisites/iot_config.sh_template

```diff
@@ -92,6 +92,10 @@ export CUSTOM_ALLOCATION_POLICY_WEBHOOK=""
 # Only the hostname is needed, not the full connection string
 export FAR_AWAY_IOTHUB_HOSTNAME=""
 
+# Storage account integration
+# Used for IoT Hub features that integrate with a storage account, e.g. DeviceImport and DeviceExport jobs
+export STORAGE_ACCOUNT_CONNECTION_STRING=""
```

Comment:
Is this environment variable defined in our build/release pipeline yaml files as well?

Reply:
Not yet. I'll work on that before merging.

Reply:
done